test/babel-plugin-istanbul.js

```diff
@@ -89,83 +87,6 @@ describe('babel-plugin-istanbul', function () {
     })
   })
 
-  // Regression tests for https://github.com/istanbuljs/babel-plugin-istanbul/issues/7
-  if (!/^v0\.10/.test(process.version)) { // mock-fs does not work on Node 0.10.
```

Comment:
I like the fact that we can get rid of this branch in the tests now, but how slow is it to have these?